mmdet/apis/inference.py

```diff
@@ -54,6 +54,12 @@ def init_detector(config, checkpoint=None, device='cuda:0', cfg_options=None):
     model.cfg = config  # save the config in the model for convenience
     model.to(device)
     model.eval()
+
+    if device == 'npu':
+        from mmcv.device.npu import NPUDataParallel
+        model = NPUDataParallel(model)
+        model.cfg = config
```

Comment:
model.to(device);model.eval() are these two lines unnecessary?

Reply:
Yes, because the model has already been set, there is no need to set it again. This place uses NPUDataParallel mainly to put input on the device